docs/framework/release-notes/release-notes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: .NET Framework release information
+description: Improvements in the .NET Framework servicing cumulative updates.
+ms.date: 01/14/2025
+---
+# .NET Framework Release Notes
+
+_Released January 14, 2025_
```

Comment:
It seems odd to include a release date in an index file.

Reply:
I wondered about that also.  I thought it was required to have it though, so I left it.  I'll remove this date.